site/public/js/ta-grading.js

```diff
@@ -746,6 +746,9 @@ function downloadFile(html_file, url_file) {
     else if (url_file.includes("results")) {
         directory = "results";
     }
+    else if (url_file.includes("checkout")) {
```

Comment:
What is the contents of url_file?  Is it the path to a students file?
If the student names a file in their repository "my_src/my_submissions_results.cpp" for example
will this code break because url_file will be "checkout/my_src/my_submissions_results.cpp"?

Should this be a more specific check so we don't get confused by the file name?

Reply:
True. This is too crude a way to locate a directory. Will fix it.